projects/packages/scheduled-updates/src/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-update-schedules.php

```diff
@@ -397,6 +385,59 @@ public function prepare_item_for_response( $item, $request ) {
 		return rest_ensure_response( $item );
 	}
 
+	/**
+	 * Checks that the "plugins" parameter is not empty.
+	 *
+	 * @param array           $plugins List of plugins to update.
+	 * @param WP_REST_Request $request Request object.
+	 * @param string          $param   The parameter name.
+	 * @return bool|WP_Error
+	 */
+	public function validate_plugins_param( $plugins, $request, $param ) {
+		$result = rest_validate_request_arg( $plugins, $request, $param );
+		if ( is_wp_error( $result ) ) {
+			return $result;
+		}
+
+		$installed_plugins = array_filter( $plugins, array( $this, 'is_plugin_installed' ) );
+		if ( empty( $installed_plugins ) ) {
+			add_filter( 'rest_request_after_callbacks', array( $this, 'transform_error_response' ) );
+
+			return new \WP_Error(
+				'rest_forbidden',
+				__( 'None of the specified plugins are installed.', 'jetpack-scheduled-updates' ),
+				array( 'status' => 400 )
+			);
+		}
+
+		$unmanaged_plugins = array_diff( $plugins, array_filter( $plugins, array( Scheduled_Updates::class, 'is_managed_plugin' ) ) );
+		if ( empty( $unmanaged_plugins ) ) {
+			add_filter( 'rest_request_after_callbacks', array( $this, 'transform_error_response' ) );
+
+			return new \WP_Error(
+				'rest_forbidden',
+				__( 'All of the specified plugins are managed.', 'jetpack-scheduled-updates' ),
```

Comment:
Can we think of a better error message?

Reply:
How about: The specified plugins are managed and auto-updated by WordPress.com. To schedule updates, install additional plugins from the WordPress.com Marketplace.